docs/howto/create-a-release.md

```diff
@@ -53,6 +53,19 @@
    rm -r dist
 ```
 
+## Test the docs
+
+For any projects which are auto-documented by cylc-doc, currently:
+
+* cylc-flow
+
+Ensure the docs build against master by manually triggering the test workflow
+in cylc-doc.
```

Comment:
Can you put a tad more detail in for the unfamiliar?
Where and how do I do this?

Reply:
I'm reluctant to document how to use GH actions, partly because GH does a better job of documenting this but also as the instructions will continue to change, only a few weeks ago the actions tab was in a completely different place and workflow_dispatch didn't even exist.